plugins/grafana-custom-plugins/grafana-http-table-plugin/src/plugin.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
+  "type": "panel",
+  "name": "grafana-http-table-plugin",
+  "id": "theia-grafana-http-table-plugin",
+  "info": {
+    "keywords": ["panel"],
+    "description": "",
+    "author": {
+      "name": "Antrea io"
+    },
+    "logos": {
+      "small": "img/logo.svg",
```

Comment:
please follow https://github.com/antrea-io/theia/blob/main/plugins/grafana-custom-plugins/grafana-chord-plugin/src/plugin.json#L14

Reply:
not solved yet